### PR TITLE
Stabalizations related to reset commands

### PR DESCRIPTION
**Pull request by MiccWan**
#484

### CODE REVIEW

File: dbux-code/src/projectViews/practiceView/ProjectNode.js [hunk 1]
```diff
@@ -59,27 +59,24 @@ export default class ProjectNode extends BaseTreeViewNode {
...
-            progress.report({ message: 'Done.' });
           }, {
             cancellable: false,
-            location: ProgressLocation.Notification,
             title: this.project.name,
           });
         }
       };
       await showInformationMessage(confirmMessage, btnConfig, { modal: true });
+      await showInformationMessage('Project has been deleted successfully.');
```
Comment (Domiii) [inline dbux-code/src/projectViews/practiceView/ProjectNode.js:79]: @MiccWan This should go into the `Ok` function, no?

File: dbux-projects/src/ProjectsManager.js [hunk 2]
```diff
@@ -356,19 +355,20 @@ export default class ProjectsManager {
    * @param {Bug} bug 
    */
   async resetBug(bug) {
-    try {
-      await bug.project.gitResetHard(true, 'This will discard all your changes on this bug.');
+    const confirmMessage = 'This will discard all your changes on this bug. Are you sure?';
```
Comment (Domiii) [inline dbux-projects/src/ProjectsManager.js:358]: Probably want to `deactivateBug()` first?

Comment (MiccWan) [reply dbux-projects/src/ProjectsManager.js:358]: use `ProjectManager.stopPractice` for consistency

File: dbux-code/src/projectViews/practiceView/ProjectNode.js (hunk 1 above)
Comment (MiccWan) [reply dbux-code/src/projectViews/practiceView/ProjectNode.js:79]: I'm so stupid :P

**Pull request merged**